src/main/java/com/strandls/utility/dao/TagsDao.java

```diff
@@ -94,4 +94,31 @@ public List<Tags> fetchNameByLike(String phrase) {
 
 	}
 
+	@SuppressWarnings("unchecked")
+	public Tags fetchTag(String phrase) {
+		Session session = sessionFactory.openSession();
+		Tags tags = null;
+		List<Object[]> result = null;
+
+		String qry = "SELECT id, name FROM public.tags where name = :phrase";
+
+		try {
+			Query<Object[]> query = session.createNativeQuery(qry);
+			query.setParameter("phrase", phrase);
+			result = query.getResultList();
+
+			if (!result.isEmpty()) {
+				Object[] obj = result.get(0);
+				tags = new Tags(Long.parseLong(obj[0].toString()), obj[1].toString());
```

Comment:
It is possible to do it ,but if we look into this file we have similar approach used by all other functions , if we change this one it will look different form others , so i think we should leave this as it is to maintain uniformity .